modules/closure-templates/1.0.1/MODULE.bazel

```diff
@@ -0,0 +1,173 @@
+module(
+    name = "closure-templates",
+    version = "1.0.1",
+    compatibility_level = 1,
+)
+
+# -----------------------------------------------------------------
+# bazel deps
+# -----------------------------------------------------------------
+
+bazel_dep(name = "protobuf", version = "32.1", repo_name = "com_google_protobuf")
+bazel_dep(name = "rules_java", version = "8.16.1")
+bazel_dep(name = "rules_jvm_external", version = "6.8")
+bazel_dep(name = "rules_proto", version = "7.1.0")
+
+# -----------------------------------------------------------------
+# jvm_import_external deps
+# -----------------------------------------------------------------
+
+jvm_import_external = use_repo_rule("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_import_external")
```

Comment:
out of curiosity, is the usage of `jvm_import_external` necessary because `rules_jvm_external` is lacking some feature?

I'm asking since `jvm_import_external` is one of the last few JVM-related repo rules shipped with `bazel_tools`, and I had been looking at removing it a while ago for Bazel 9.0 (I didn't end up having time for it).